jstests/tests/home/test_configurables.js

```diff
@@ -1,26 +1,17 @@
 define([
+    "jquery",
     "home/configurables"
-], function (configurables) {
+], function ($, configurables) {
```

Comment:
Here I use jQuery for the `extend` method which does a deep copy of the dictionary representing the model of the configurable. Underscore provides an `extend`method too but it doesn't do a deep copy of the dictionary.
Instead of returning a list of dictionaries representing the models of configurables in `configurable.js`I could return a list of classes. So we get rid of this deep copy of dictionaries.